portafly/doc/ux_specs/index.md

```diff
@@ -22,6 +22,7 @@ Please refer to the single page files together with mockups linked in each page
 ##### Product
 * Applications
   * [Applications index](./product/applications/index.md)
+* [Overview](./product/index.md)
```

Comment:
Do we want the indexes in the same order as in the sidebar navigation? If so I guess Overview goes first.
```suggestion
* [Overview](./product/index.md)
* Applications
  * [Applications index](./product/applications/index.md)
```

Reply:
Updated